<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DummyModelPanel</class>
 <widget class="QWidget" name="DummyModelPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>264</width>
     <height>304</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="logLabel"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="clearLogButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="autoSaveCheckBox">
      <property name="text">
       <string>Auto save</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="saveSnapshotButton">
      <property name="text">
       <string>Save snapshot</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="loadSnapshotButton">
      <property name="text">
       <string>Restore last snapshot</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="controlsArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>555</width>
      <height>335</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>535</width>
      <height>335</height>
     </size>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>461</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="controlsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
